src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java

```diff
@@ -58,6 +60,22 @@ public String getSettingValue(String settingName, String defaultValue) {
         return settings.getProperty(settingName, defaultValue);
     }
 
+    private static void loadHttpsEnabled(PluginSettings instance, Properties settings) throws Exception {
+        String httpsEnabledString = settings.getProperty(HTTPS_ENABLED, "False");
+        if ( httpsEnabledString == null ) {
```

Comment:
nit: remove extra space between '(' and httpsEnabledString

Reply:
Done